speakers/src/main/java/in/droidcon/speakers/presentation/ui/SpeakersAdapter.kt

```diff
@@ -1,8 +1,7 @@
 package `in`.droidcon.speakers.presentation.ui
 
-import `in`.droidcon.data.speakers.model.SpeakerEntity
+import `in`.droidcon.speakers.model.SpeakerEntity
 import `in`.droidcon.speakers.R
-import `in`.droidcon.speakers.model.SpeakerItem
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
```

Comment:
**NIT** I know this is not a part of this PR but the findViewById used here could be easily replaced by using kotlin synthetic like below,

```
with(itemView) {
   speakerImgView...
}
```

Reply:
Left it, as not part of this PR, also not sure, but we might change the adapter future, since the UI implementation isn't complete and as par zeplin yet.

Reply:
Done